packages/block-editor/src/components/button-block-appender/style.scss

```diff
@@ -35,7 +35,9 @@
 // When the appender shows up in empty container blocks, such as Group and Columns, add an extra click state.
 .block-list-appender:only-child {
 	.is-layout-constrained.block-editor-block-list__block:not(.is-selected) > &,
-	.is-layout-flow.block-editor-block-list__block:not(.is-selected) > & {
+	.is-layout-flow.block-editor-block-list__block:not(.is-selected) > &,
+	.is-layout-constrained.block-editor-block-list__block:not(.is-selected) .block-editor-block-list__layout > &,
+	.is-layout-flow.block-editor-block-list__block:not(.is-selected) .block-editor-block-list__layout > & {
```

Comment:
I had to add these styles so that empty Group blocks in classic themes, where the block has an inner wrapper, also show the non-selected state. Cc @jasmussen in case you have better ideas on how to address this!

Reply:
I'm unable to test at the moment, but the main change to consider would be adding a `>` between the selectors so it only targets the direct descendant, and not any nested groups. Not sure if necessary in practice since the group would by definition be empty and not have anything inside to inherit. 🤔 

Though to finish the thought, this would be the alternative:

```
	.is-layout-constrained.block-editor-block-list__block:not(.is-selected) > .block-editor-block-list__layout > &,
	.is-layout-flow.block-editor-block-list__block:not(.is-selected) > .block-editor-block-list__layout > & {
```

And the way to test would be to add an empty group inside a group. As noted, probably no difference to spot.

Reply:
Good point, as it is it could potentially make a placeholder appear in any child block with only an inserter. I'll update it, thanks!